.coveragerc

```diff
@@ -1,7 +0,0 @@
-[run]
```

Comment:
This change seems unrelated. Please don't include it in this PR.

Reply:
Done, it was removed accidentally